src/css/screen.scss

```diff
@@ -46,7 +46,8 @@
 	"logic/variables.font-tracking",
 	"logic/variables.font-weights",
 	"logic/variables.line-heights",
-	"logic/variables.global";
+	"logic/variables.global",
+	"logic/variables.font-sizes";
```

Comment:
Could you move this to above line 46 to put it in alphabetical order? So:

```
...
"logic/variables.font-families",
"logic/variables.font-sizes"
"logic/variables.font-tracking",
"logic/variables.font-weights",
...
```

Reply:
Cool, done. Thanks